assets/queries/terraform/alicloud/oss_bucket_allows_all_actions/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "ec62a32c-a297-41ca-a850-cab40b42094a",
+  "queryName": "OSS Bucket Allows All Actions",
+  "severity": "HIGH",
+  "category": "Access Control",
+  "descriptionText": "OSS Buckets should not allow all actions from all principals.",
```

Comment:
```suggestion
  "descriptionText": "OSS Buckets should not allow all actions (wildcard) from all principals, as to prevent leaking private information to the entire internet or allow unauthorized data tampering/deletion. This means the 'Effect' must not be 'Allow' when the 'Action' is *, for all Principals.",
```